src/Exchange.ts

```diff
@@ -268,7 +268,11 @@ export class Exchange {
         ecSignature.s
       ], {
         gasPrice,
-        gasLimit: gasLimit || (takerPathObj.estimatedGas + makerPathObj.estimatedGas) * 1.5
+        gasLimit: gasLimit ||
+          new BigNumber(takerPathObj.estimatedGas)
+            .plus(new BigNumber(makerPathObj.estimatedGas))
+            .multipliedBy(1.5)
+            .toString()
```

Comment:
I think we should use BigNumbers internally and only convert it to a string when exporting it out of the lib. That means we can always do calculation. Otherwise we always need to convert.